modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -269,177 +383,212 @@ The output from this command lists the value for the key:
 
-To view data in the Object Store:
+To test your app:
 
-. Log into Anypoint Platform and click *Runtime Manager*.
-. Click the name of your application to view the application's dashboard.
-. Click *Object Store* from the left navigation bar:
+. In *Runtime Manager*, click *Applications* and ensure that the `osv2-demo-studio-6` app is running.
+. Click the status for the `osv2-demo-studio-6` app to display its details in the right pane.
+. Copy the *App url* value:
++
+.The arrow shows *App url* in the details pane in Runtime Manager.
+image::os-tutorial-rtm-deploy-url-3.png[App url in Runtime Manager]
+. Run the following `curl` command from a command prompt.
++
+Replace _APP-URL_ with the *App url* value that you copied from Runtime Manager. 
++
+[source,console]
+----
+$ curl -X POST -H "Content-Type: application/json" -d '{ "key": "myStudio6Key", "value": "OSv2 Test from Studio 6" }' "http://APP-URL/store";echo
+----
++
+Each time you submit this command, change the key name so that each key is unique.
++
+The output is an echo:
++
+[source,console]
+----
+KEY: myStudio6Key AND VALUE: OSv2 Test from Studio 6 WERE SAVED
+----
 +
-image::osv2-in-nav-bar.png[]
+. In Runtime Manager, in the details pane, click *Manage application*.
 +
-The Object Store user interface:
+.The arrow shows *Manage application* in the details pane in Runtime Manager.
+image::os-tutorial-rtm-manage-app-3.png[Manage application in the details pane]
+. Click *Object Store* in the left navigation pane.
+. Click the object store, the partition, and then the key:
 +
-image::osv2-ui.png[]
+.The screenshot shows (*1*) the *Object Store* tab, (*2*) the key, and (*3*) the value in the Object store.
+image::os-tutorial-rtm-key-in-os-3.png[Object Store tab, key, and value in Runtime Manager]
+. Hover over the *...* icon to display actions for each object:
 +
-* The default object store name is `DEFAULT_USER_STORE`.
-* The columns show the object store name, partition name, key, and key data. The partition groups object store keys.
+.The screenshot shows (*1*) the *Metadata* and (*2*) the *Delete Value* actions in the Object store.
+image::os-tutorial-rtm-os-actions-3.png[]
 +
-. Click the object store name, partition name, and key to view each item's value.
-. To delete a key, hover over a key name, and click the trash can icon. Similarly,
-you can delete a partition or the store itself by hovering and clicking the trash
-can icon.
+* To delete an object store, partition, key, or value, hover over the *...* icon and click the trash can icon.
+* To display the key value and any metadata associated with the value, hover over the *...* icon and click the metadata icon.
 
 
 == See Also
 
+* xref:runtime-manager::deploying-to-cloudhub.adoc[Deploying to CloudHub]
+* xref:runtime-manager::runtime-manager-switch-env.adoc[Using Different Environments in Runtime Manager]
```

Comment:
I've been bad about editing gerunds out of titles, but we need to start doing it going forward.

Reply:
Those are concepts. Should we change the link text for xrefs?